pom.xml

```diff
@@ -107,9 +107,9 @@
     </build>
 
     <properties>
-        <revision>0.1.5</revision>
+        <revision>0.2.0</revision>
```

Comment:
This should be a minor version bump, since its a bugfix. Refer to semver for guidance.

Reply:
This is also JDK17 compliant change. 
Hence the bump to major version seemed relevant.

Reply:
Reducing to a minor version bumnp only, as we will have to make a separate pr (and maybe a separate pom) that compiles using java 17 to qualify for a mjor version bump